models/reviews.model.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection.js")
+
+exports.fetchReview = (reviewId) => {
+    if(!/^\d+$/.test(reviewId)) {
```

Comment:
Great use of regex here to check whether the reviewId is a number. PSQL will actually throw an automatic error for us if the id is invalid, so it's actually not needed!